.travis.yml

```diff
@@ -50,16 +49,11 @@ install:
 script:
   - export PIPENV_IGNORE_VIRTUALENVS=1
   # for some reason the migrations check is failing only on django 2.0
-  - if [[ $TRAVIS_PYTHON_VERSION == 3.5 ]]; then pipenv run ./tests/manage.py makemigrations config pki --dry-run | grep "No changes detected"; fi
-  - pipenv run coverage run --source=openwisp_controller runtests.py
   - |
-    if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
-      # gets commit message of last commit before pull request merge
-      COMMIT_MESSAGE=$(git log $TRAVIS_PULL_REQUEST_SHA --format=%B -n 1)
-      printf "Checking commit message:\n\n"
-      printf "$COMMIT_MESSAGE\n\n"
-      pipenv run checkcommit --message "$COMMIT_MESSAGE"
+    if [[ $TRAVIS_PYTHON_VERSION == 3.5 ]]; then
+      pipenv run openwisp-utils-qa-checks --migration-path "./openwisp_controller/config/migrations ./openwisp_controller/connection/migrations ./openwisp_controller/geo/migrations ./openwisp_controller/pki/migrations" --migration-module "config pki" --migrations-to-ignore "12 0 0 4"
```

Comment:
I would move `--migration-module` after `--migrations-to-ignore` for clarity.
Can you use a multiline statement to make ti more readable pelase?

Also, `--migration-module` should be:

```
--migration-module "config connection geo pki"
```